### ISSUE TITLE
Server owned :(

### ISSUE THREAD

**Issue by beldini** [MinetestForFun/server-minetestforfun-skyblock#216]
At about 18:45 AUCST (9:15 UTC I think). 

My wife had joined me to play on the server. All of a sudden she was booted with an access denied message along the lines of:

Access Denied: Kicked: Worldedit is broken, fix it!

(Or something to that effect, sorry we didnt copy it down).

The the server crashed and we are all getting a message:

Access Denied: Reason: You are using an unofficial client. Use the official client from minetest.org.  

(Which is very obviously NOT the official site)

**Comment by aerozoic** [comment]
@Darcidride 
```
2017-05-14 13:57:42: ACTION[Server]: Server: Player with the name "cessna151" tried to connect from ???.???.???.??? but it was disallowed for the following reason: You are using an unofficial client. Use the official client from minetest.org
2017-05-14 13:58:12: ACTION[Server]: Server: Player with the name "beldin" tried to connect from ???.???.???.??? but it was disallowed for the following reason: You are using an unofficial client. Use the official client from minetest.org
```
:large_orange_diamond:

**Comment by aerozoic** [comment]
Shivajiva says this was a hack and World Edit needs updated.
:large_orange_diamond:

**Comment by aerozoic** [comment]
fixed:  26c7767
:large_orange_diamond:
